salt/metalk8s/kubernetes/etcd/installed.sls

```diff
@@ -1,22 +1,23 @@
-{% from "metalk8s/registry/macro.sls" import build_image_name with context %}
+{%- from "metalk8s/registry/macro.sls" import build_image_name with context %}
 
-{% set image_name = build_image_name('etcd', '3.2.18') %}
+include:
+  - metalk8s.kubernetes.ca.etcd.advertised
+  - .certs
 
-{% set host_name = salt.network.get_hostname() %}
-{% set host = grains['metalk8s']['control_plane_ip'] %}
+{%- set host_name = salt.network.get_hostname() %}
```

Comment:
Not for this PR, but: this shouldn't be the hostname but the node/minion id (which can be, but is not necessarily, the hostname)